docs/src/user-guide/getting-started.md

```diff
@@ -93,6 +93,57 @@ Your `.eslintrc.{js,yml,json}` configuration file will also include the line:
 
 Because of this line, all of the rules marked "(recommended)" on the [rules page](/docs/rules) will be turned on.  Alternatively, you can use configurations that others have created by searching for "eslint-config" on [npmjs.com](https://www.npmjs.com/search?q=eslint-config).  ESLint will not lint your code unless you extend from a shared configuration or explicitly turn rules on in your configuration.
 
+## Global Install
+
+It is also possible to install ESLint globally, rather than locally, using `npm install eslint --global`. However, this is not recommended, and any plugins or shareable configs that you use must still be installed locally if you install ESLint globally.
+
+## Manual Set Up
+
+You can also manually set up ESLint in your project.
+
+Before you begin, you must already have a `package.json` file. If you don't, make sure to run `npm init` or `yarn init` to create the file beforehand.
+
+1. Install the ESLint package in your project:
+
+   ```shell
+   npm install --save-dev eslint
+   ```
+
+1. Add an `.eslintrc` file in one of the [supported configuration file formats](./configuring/configuration-files#configuration-file-formats).
+
+   ```shell
+   # Create JavaScript configuration file
+   touch .eslintrc.js
+   ```
+
+1. Add configuration to the `.eslintrc` file. Refer to the [Configuring ESLint documentation](configuring/) to learn how to add rules, environments, custom configurations, plugins, and more.
```

Comment:
I feel like this line has too much `to` personally 😅  is that fine? or should we rephrase this?

Reply:
i think it's fine personally.